bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -441,6 +453,31 @@ spec:
                   requests:
                     cpu: 20m
                     memory: 300Mi
+              - name: kube-rbac-proxy
+                image: quay.io/openshift/origin-kube-rbac-proxy:latest
```

Comment:
This is being pulled from the internet at runtime. We cannot do this.
Is a build of this image being shipped as part of the payload?